run/opencl/keepass_kernel.cl

```diff
@@ -1,14 +1,26 @@
 /*
- * This software is Copyright (c) 2018 magnum,
+ * This software is Copyright (c) 2018-2021 magnum,
  * and it is hereby released to the general public under the following terms:
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted.
  */
+
+#if MAX_CONT_SIZE >= 0xff00
```

Comment:
What's `CONT` here, isn't it a typo of `CONST`? (I see it's moved from below in this file, rather than introduced with this PR, but anyway.)

Reply:
I think it's an odd abbreviation for content. I'll leave it alone.